src/api/layers/WMSCapabilitiesParser.class.js

```diff
@@ -42,6 +42,43 @@ export default class WMSCapabilitiesParser {
         this.originUrl = new URL(originUrl)
     }
 
+    /**
+     * Returns information about where feature identification can be accessed, or null if this WMS
+     * has no such capabilities
+     *
+     * @param {boolean} ignoreError
+     * @returns {null | ExternalLayerGetFeatureInfoCapability}
+     * @see https://www.mediamaps.ch/ogc/schemas-xsdoc/sld/1.2/capabilities_1_3_0_xsd.html#Capability
+     */
+    getFeatureInfoCapability(ignoreError = true) {
+        if (Array.isArray(this.Capability?.Request?.GetFeatureInfo?.DCPType)) {
+            const httpElement = this.Capability.Request.GetFeatureInfo?.DCPType[0].HTTP
+            let baseUrl = null
+            let method = 'GET'
+            if (httpElement?.Get) {
+                baseUrl = httpElement.Get.OnlineResource
+            } else if (httpElement?.Post) {
+                method = 'POST'
+                baseUrl = httpElement.Post.OnlineResource
+            } else {
+                log.error(
+                    "Couldn't parse GetFeatureInfo data",
+                    this.Capability.Request.GetFeatureInfo
+                )
+                if (ignoreError) {
+                    return null
+                }
+                throw new CapabilitiesError('Invalid GetFeatureInfo data')
```

Comment:
Here I would add a translation key after the message

```javascript
throw new CapabilitiesError('Invalid GetFeatureInfo data', 'invalid_get_feature_info')
```

We could use it then later for displaying an error